apps/vmq_passwd/c_src/Makefile

```diff
@@ -3,7 +3,7 @@ CFLAGS_EXTRA =
 
 ifeq ($(shell uname -s), Darwin)
 	# OSX with homebrew
-	OPENSSL_DIR ?= /usr/local/opt/openssl
+	OPENSSL_DIR = $(brew --prefix openssl)
```

Comment:
We should keep the ?= to support osx without homebrew

Reply:
@codeadict thank you, PR updated

Reply:
@bopbi - See comment below, we should be able to merge after that with @ioolkos approval